Project1/calculator.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+#include <iostream>
+#include <string>
+
+enum class CalculatorException{
+	DIVIDE_BY_ZERO,
+	UNKNOWN_OPERATION
+};
+
+class Calculator {
```

Comment:
Do we allow people to inherit from Calculator?
Do we want them to do it?
If not - use the 'final' specifier.